tests/integration/test_lstm_backward_test.py

```diff
@@ -79,24 +80,24 @@ def test_basic():
         dC_num[t] += (1 - tanhCt_num**2) * (
             IFOGf_num[t, :, 2 * d : 3 * d] * dHout_num[t]
         )
-        # assert num.allclose(dC_np[t], dC_num[t])
+        # assert allclose(dC_np[t], dC_num[t])
 
         if t > 0:
             dIFOGf_np[t, :, d : 2 * d] = C_np[t - 1] * dC_np[t]
             dIFOGf_num[t, :, d : 2 * d] = C_num[t - 1] * dC_num[t]
-            # assert num.allclose(dIFOGf_np[t,:,d:2*d], dIFOGf_num[t,:,d:2*d])
+            # assert allclose(dIFOGf_np[t,:,d:2*d], dIFOGf_num[t,:,d:2*d])
```

Comment:
What is this `IFOGf` abbreviation that I see in a few tests?

Reply:
`IFOG = input, forget, output, gate`, which refer to [LSTM](https://en.wikipedia.org/wiki/Long_short-term_memory) components.

Probably just an artifact of the original code the tests were based on https://gist.github.com/karpathy/587454dc0146a6ae21fc#file-gistfile1-py-L39-L40.